src/target-sc/ilator.cc

```diff
@@ -623,6 +656,7 @@ bool Ilator::GenerateBuildSupport(const std::string& dir) {
       "project({project} LANGUAGES CXX)\n"
       "\n"
       "option(ILATOR_VERBOSE \"Enable instruction sequence logging\" OFF)\n"
+      // "option(ILATOR_INSN_VERBOSE \"Enable instruction state updates logging\" OFF)\n"
```

Comment:
Seems like the flag `ILATOR_INSN_VERBOSE` is not used. Should we keep this?

Reply:
We can delete it.
I was thinking setting different level of debug logging, and want to separate them. But it seems that combining them works fine for my current examples.